projects/plugins/wpcomsh/composer.json

```
@@ -0,0 +1,141 @@
+		"automattic/text-media-widget-styles": "^2.0",
+		"automattic/wc-calypso-bridge": "2.5.1",
+		"wordpress/classic-editor-plugin": "1.5",
+		"automattic/jetpack-config": "@dev",
+		"automattic/jetpack-post-list": "@dev",
+		"automattic/jetpack-mu-wpcom": "@dev",
+		"tubalmartin/cssmin": "^4.1"
+	},
+	"require-dev": {
+		"automattic/jetpack-changelogger": "@dev",
+		"automattic/wordbless": "0.4.2",
+		"dealerdirect/phpcodesniffer-composer-installer": "^1.0",
+		"phpunit/phpunit": "^9",
+		"yoast/phpunit-polyfills": "1.1.0",
+		"wp-coding-standards/wpcs": "^3.0"
+	},
+	"autoload": {
+		"classmap": [
+			"block-theme-footer-credits",
+			"customizer-fixes",
+			"custom-colors",
+			"endpoints",
+			"feature-plugins",
+			"footer-credit",
+			"frontend-notices",
+			"imports",
+			"jetpack-require-connection-owner",
+			"jetpack-token-error-header",
+			"jetpack-token-resilience",
+			"lib",
+			"logo-tool",
+			"mailpoet",
+			"notices",
+			"privacy",
+			"private-site",
+			"safeguard",
+			"share-post",
+			"storage",
+			"storefront",
+			"widgets",
+			"wpcom-features",
+			"wpcom-migration-helpers",
+			"wpcom-plugins",
+			"wpcom-themes"
+		]
+	},
+	"scripts": {
+		"phpunit": [
+			"./vendor/phpunit/phpunit/phpunit --colors=always"
+		],
+		"test-php": [
+			"@composer phpunit"
+		],
+		"post-install-cmd": "WorDBless\\Composer\\InstallDropin::copy",
+		"post-update-cmd": "WorDBless\\Composer\\InstallDropin::copy"
+	},
+	"repositories": [
+		{
+			"type": "path",
+			"url": "../../packages/*",
+			"options": {
+				"monorepo": true
+			}
+		},
+		{
+			"type": "vcs",
+			"no-api": true,
+			"url": "https://github.com/Automattic/custom-fonts.git"
+		},
+		{
+			"type": "vcs",
+			"no-api": true,
+			"url": "https://github.com/Automattic/custom-fonts-typekit.git"
+		},
+		{
+			"type": "vcs",
+			"no-api": true,
+			"url": "https://github.com/automattic/at-pressable-podcasting.git"
+		},
+		{
+			"type": "vcs",
+			"no-api": true,
+			"url": "https://github.com/Automattic/text-media-widget-styles.git"
+		},
+		{
+			"type": "vcs",
+			"no-api": true,
+			"url": "https://github.com/Automattic/wc-calypso-bridge.git"
+		},
+		{
+			"type": "package",
+			"package": {
+				"name": "wordpress/classic-editor-plugin",
+				"version": "1.5",
+				"dist": {
+					"url": "https://downloads.wordpress.org/plugin/classic-editor.1.5.zip",
+					"type": "zip"
+				},
+				"source": {
+					"url": "https://plugins.svn.wordpress.org/classic-editor/",
+					"type": "svn",
+					"reference": "tags/1.5/"
+				},
+				"autoload": {
+					"classmap": []
+				}
+			}
+		}
+	],
+	"minimum-stability": "dev",
+	"prefer-stable": true,
+	"config": {
+		"allow-plugins": {
+			"dealerdirect/phpcodesniffer-composer-installer": true,
+			"composer/installers": true,
+			"roots/wordpress-core-installer": true
+		},
+		"autoloader-suffix": "26841ac2064774301cbe06d174833bfc_wpcomshⓥ3_22_6_alpha"
+	},
+	"extra": {
+		"mirror-repo": "Automattic/wpcom-site-helper",
```

We probably want to enable autorelease and autotagger.
```suggestion
		"mirror-repo": "Automattic/wpcom-site-helper",
		"autorelease": true,
		"autotagger": true,
```
Possibly also Jetpack Beta
```suggestion
		"mirror-repo": "Automattic/wpcom-site-helper",
		"autorelease": true,
		"autotagger": true,
		"beta-plugin-slug": "wpcomsh",
```